README.md

```diff
@@ -66,9 +66,9 @@ Detox is built from the ground up to support React Native projects as well as pu
 
 The following React Native versions have been tested:
 
-| iOS     | Android                                                                                                           |
-|---------|-------------------------------------------------------------------------------------------------------------------|
-| ≤0.66.x | ≥0.59, ≤0.66 - Visibility edge-case: see this [RN issue](https://github.com/facebook/react-native/issues/23870)\* |
+| iOS                                                                                            | Android                                                                                                                                                                                                                                        |
+|------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ≤0.66.x                                                                                        | ≥0.64, ≤0.66 - <ul> <li>Visibility edge-case: see this [RN issue](https://github.com/facebook/react-native/issues/23870) </li><li>Switch edge-case: see this [RN issue](https://github.com/facebook/react-native/issues/32594) </li></ul> |
```

Comment:
I don't think the switch edge-case should be mentioned here, because we've managed to work our way around it.